web/themes/joinup/templates/content/node--event--full.html.twig

```diff
@@ -10,25 +10,39 @@
 <article{{ attributes }}>
   <div class="page__content">
     {{ content.body }}
-    {{ content.field_event_spatial_coverage }}
```

Comment:
Should be removed from display when not used in the template.

Reply:
Ok, I removed spatial coverage from the default display.